pyrobosim/examples/demo_rrt.py

```diff
@@ -15,15 +15,22 @@
 
 def test_rrt():
     """Creates an RRT planner and plans"""
-    rrt = RRTPlanner(world, bidirectional=True, rrt_connect=False, rrt_star=True)
+    planner_config = {
```

Comment:
Works, but the default `compress_path` option makes this look weird. I think it should be off by default, whereas I see that now this option is constantly hard-coded as true.

Furthermore, I would offer the suggestion to remove `compress_path` from all the planners, and instead offer it as a post-processing utility that people can opt into. Again, defaulting to false because I want to make sure that people see the "original" planner output by default for educational reasons.

Reply:
This is actually not hard-coded.
If you pass "compress_path: False" in the config, it will no longer compress the path.

Reply:
Oh, right, that just has the defaults. My bad. Well, then just changing the default to True will work then :)